README.md

```diff
@@ -1,69 +1,79 @@
 ## Segmentation of flights during ORCESTRA
 
-The Research Flights (RFs) during ORCESTRA can be divided into different segments.
-For example circles and straight legs were purposefully conducted maneuvers during which
-a distinct sampling behaviour of the various instruments can be assumed. For future analyses
-based on specific kinds of flight segments (e.g. only based on circles) it is desired to use a
-common set of start- and end-times to assure consistency between the studies. This repository
-provides files listing the start- and end-times of  flight segments for selected platforms and 
-for each flight. Further information on the respective flight can be found in the
-[flight reports](https://github.com/orcestra-campaign/book/tree/main/orcestra_book/reports).
-
-Flight segments need not be unique or complete: a given time during a flight may belong to
-any number of segments or none at all. Segments may overlap (i.e. a segment of kind `straight_leg`
-may include a segments of kind `ec_underpass`). This allows flights to be segmented
-in multiple ways and at multiple levels of granulatiry.
+The PERCUSION research flights conducted during ORCESTRA can be divided into 
+segments. A segment is a continuos interval of the flight path that has a set 
+of associated characteristics, such as flight manuevers, Earthcare underpasses,
+or instrument calibration procedures carried during the segment. 
 
-The segmentation copies and adapts many ideas from the [flight segmentation](https://github.com/eurec4a/flight-phase-separation) conducted for the EUREC4A field campaign.
+Flight segments need not be unique nor complete: a given time during a flight 
+may belong to any number of segments or none at all. Segments may overlap, 
+allowing for flights to be segmented in multiple ways and at multiple levels of 
+granularity.
 
-### Common vocabulary - broad sampling strategy
-Following are names of segments assembled from a range of platforms describing broad sampling strategies. Many will contain subsets
-e.g. `cloud` segments will likely contain `subcloud layer` and `cloud layer` segments.
-Data providers are encouraged to use these names, especially those in the first, where applicable.
-Feel free to add to the list (e.g. with a Github pull request) if needed.
-* ground
-* transit
-* circle
-* cloud
-* calibration
-* profile
-* axbt
-* rectangle
-* sawtooth
-* racetrack
+The characteristics that can be associated with a segment are clearly defined, 
```

Comment:
Thanks for the comment Toby! I agree "clearly defined" is not what we need. I was trying to express the idea that these should be agreed upon as useful for those using the data, so that it is clear what they represent.